ament_copyright/ament_copyright/parser.py

```diff
@@ -166,7 +166,7 @@ def search_copyright_information(content):
     year = '\d{4}'
     year_range = '%s-%s' % (year, year)
     year_or_year_range = '(?:%s|%s)' % (year, year_range)
-    pattern = '^[^\n\r]?\s*Copyright\s+(%s(?:,\s*%s)*)\s+([^\n\r]+)$' % \
+    pattern = '^[^\n\r]?\s*Copyright\s+(%s(?:,\s*%s)*),?\s+([^\n\r]+)$' % \
```

Comment:
Sure.  Previous to this regex change, this was accepted:

Copyright 2017 Open Source Robotics Foundation, Inc.

But this wasn't:

Copyright 2017, Open Source Robotics Foundation, Inc.

(note the comma after the year).  It is fairly minor, but I think it makes this regex a little more forgiving of the format.